src/serac/infrastructure/logger.cpp

```diff
@@ -9,12 +9,17 @@
 #include "serac/infrastructure/initialize.hpp"
 #include "serac/infrastructure/terminator.hpp"
 
+#include <fstream>
+
 namespace serac::logger {
 
 static int logger_rank = 0;
 
 int rank() { return logger_rank; }
 
+// output stream for the SLIC LogStreams that write to a file
+static std::unique_ptr<std::ofstream> logger_ofstream;
```

Comment:
Apologies if my earlier comment wasn't clear - does this need to be on the heap or can it just be
```cpp
static std::ofstream logger_ofstream;
```

Reply:
done